### PR TITLE
Bump MAX Base

### PR DESCRIPTION
**Pull request by bdwyer2**

### CODE REVIEW

**Review by autoih — DISMISSED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -20,4 +20,6 @@ COPY . /workspace
 
 EXPOSE 5000
 
+ENV LC_ALL=C
```
Comment (xuhdev) [inline Dockerfile:24]: Why do we need this?

Comment (bdwyer2) [reply Dockerfile:24]: Without this the container crashes with this error:
```
!strcmp(locale, "C"):Error:Assert failed:in file baseapi.cpp, line 209
Segmentation fault
```

https://unix.stackexchange.com/questions/87745/what-does-lc-all-c-do

Comment (xuhdev) [reply Dockerfile:24]: https://github.com/tesseract-ocr/tesseract/issues/1670#issuecomment-515324015 but we are on 4.0 :relieved:  . Perhaps adding a comment so we know we should remove this line in the future?

Comment (bdwyer2) [reply Dockerfile:24]: Something like this?

```suggestion
# this is needed for tesseract versions prior to 4.1 (https://github.com/tesseract-ocr/tesseract/issues/1670#issuecomment-515324015)
ENV LC_ALL=C
```

Comment (xuhdev) [reply Dockerfile:24]: Sure!

**Review by xuhdev — APPROVED** [review]

**Review by autoih — APPROVED** [review]

**Pull request merged**